src/sentry/integrations/vsts/integration.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import absolute_import
+from sentry import http
+from sentry.integrations import Integration, IntegrationMetadata
+from sentry.utils.pipeline import NestedPipelineView
+from sentry.identity.pipeline import IdentityProviderPipeline
+from sentry.utils.http import absolute_uri
+DESCRIPTION = """
+VSTS
+"""
+
+metadata = IntegrationMetadata(
+    description=DESCRIPTION.strip(),
+    author='The Sentry Team',
+    issue_url='https://github.com/getsentry/sentry/issues/new?title=VSTS%20Integration:%20&labels=Component%3A%20Integrations',
+    source_url='https://github.com/getsentry/sentry/tree/master/src/sentry/integrations/vsts',
+    aspects={},
+)
+
+
+class VSTSIntegration(Integration):
+    key = 'vsts'
+    name = 'VSTS'
+    metadata = metadata
+    domain = '.visualstudio.com'
```

Comment:
@EvanPurkhiser the `server:port` is for on-prem only. Since it isn't a priority, I will take that out of the description. For the off the shelf version you must have that domain.